src/OSPSuite.Core/Domain/Constants.cs

```diff
@@ -256,24 +257,20 @@ public static class Filter
          public static readonly string MATLAB_FILTER = FileFilter("Matlab�", MATLAB_EXTENSION);
          public static readonly string CSV_FILE_FILTER = FileFilter("CSV", CSV_EXTENSION);
          public static readonly string JSON_FILE_FILTER = FileFilter("Json", JSON_EXTENSION);
+         public static readonly string MARKDOWN_FILE_FILTER = FileFilter("Markdown", MARKDOWN_EXTENSION);
```

Comment:
I have added the markdown extension as well. Not required for this PR but I will need it in PKSim and Qualif runner